packages/design-system-react/src/components/avatar-network/AvatarNetwork.constants.ts

```diff
@@ -0,0 +1,16 @@
+// Remove this file if it's not needed
+export const AVATARNETWORK_CLASSMAP = {};
+
+import { AvatarBaseSize } from '../avatar-base';
+import { TextVariant } from '../text';
+
+export const AVATAR_NETWORK_SIZE_TO_TEXT_VARIANT_MAP: Record<
+  AvatarBaseSize,
+  TextVariant
+> = {
+  [AvatarBaseSize.Xs]: TextVariant.BodyXs,
+  [AvatarBaseSize.Sm]: TextVariant.BodyXs,
+  [AvatarBaseSize.Md]: TextVariant.BodySm,
+  [AvatarBaseSize.Lg]: TextVariant.BodyMd,
+  [AvatarBaseSize.Xl]: TextVariant.BodyMd,
```

Comment:
To prevent circular dependency we export AvatarBaseSize as AvatarNetworkSize but use AvatarBaseSize internally.